_data/projects/meshery.yaml

```
@@ -0,0 +1,19 @@
+---
+name: Meshery
+desc: The service mesh management plane
+site: https://meshery.io
+tags:
+- oss
+- service-mesh
+- istio
+- linkerd
+- kubernetes
+- docker
+- prometheus
+- grafana
+- consul
+- performance
+- networking
+upforgrabs:
+  name: help wanted
+  link: https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+
```

I believe this is the root cause - the stray unicode character is throwing our JSONSchema validation.

```suggestion
  link: https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+
```

---

> @leecalcote thanks for the heads up - I'm gonna see if I can fix that hint to support org-wide cheks as part of reviewing this PR
> 
> There's also a change in here:
> 
> ```
>  1 files found in projects directory which are not YAML files:
>  - _data/projects/meshery.yaml
> ```
> 
> I believe this needs to be `.yml` to work with Jekyll.

Much appreciated, @shiftkey. Do let me know if there's any further information that I can provide or updates that I should make here. 👍